components/raftstore/src/store/fsm/apply.rs

```diff
@@ -572,6 +572,24 @@ fn should_write_to_engine(cmd: &RaftCmdRequest, kv_wb_keys: usize) -> bool {
     false
 }
 
+/// Checks if a write is needed to be issued after handling the command.
+fn should_write_to_engine_after(cmd: &RaftCmdRequest) -> bool {
+    if cmd.has_admin_request() {
+        return true;
```

Comment:
Having written to the engine before, do we need to write again?

Reply:
No, it has not been written to the engine. What has been written are commands before ingest sst. So we need to write it again.

Reply:
I don't think every admin command needs to trigger a write immediately.

Reply:
It does not trigger a write immediately, instead, it just set a sync hint and performs a write at the `fn end(&self)`.